### PR TITLE
Custom karma name

### PR DESCRIPTION
**Pull request by nledez**
To solve https://github.com/prymitive/karma/issues/993

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ func init() {
 	pflag.Bool("alertmanager.proxy", false,
 		"Proxy all client requests to Alertmanager via karma (only used with simplified config)")
 
+	pflag.String("karma.name", "karma",
```
[the remark this answers is not in the document]
Comment (nledez) [reply internal/config/config.go]: I put `--karma.name` to can be "future proof”.
It’s a parameter only link to Karma instance.
In the future, another parameter can be color (for example), etc.
What do you think about that?

[the remark this answers is not in the document]
Comment (nledez) [reply internal/config/config.go]: Ho and another thing is the pflag/viper conversion of parameters `--karma.name` in `KARMA_NAME` is more readable in my point of view.

[the remark this answers is not in the document]
Comment (prymitive) [reply internal/config/config.go]: I don't think I have any strong opinions here, and `NAME` env variable would be in fact too generic and confusing, so let's go with `KARMA_NAME`

**Comment by prymitive** [comment]
Thanks!

**Comment by nledez** [comment]
You're welcome! :)

**Comment by nledez** [comment]
Last things @prymitive can you make a release (mainly for Docker)?
Thanks.

**Comment by prymitive** [comment]
I am planning on a release today, need to merge one more pr

**Pull request merged**